python/docs/source/_static/versions.json

```diff
@@ -0,0 +1,290 @@
+    },
+    {
+        "name": "2.3.3",
+        "version": "2.3.3"
+    },
+    {
+        "name": "2.3.2",
+        "version": "2.3.2"
+    },
+    {
+        "name": "2.3.1",
+        "version": "2.3.1"
+    },
+    {
+        "name": "2.3.0",
+        "version": "2.3.0"
+    },
+    {
+        "name": "2.2.3",
+        "version": "2.2.3"
+    },
+    {
+        "name": "2.2.2",
+        "version": "2.2.2"
+    },
+    {
+        "name": "2.2.1",
+        "version": "2.2.1"
+    },
+    {
+        "name": "2.2.0",
+        "version": "2.2.0"
+    },
+    {
+        "name": "2.1.3",
+        "version": "2.1.3"
+    },
+    {
+        "name": "2.1.2",
+        "version": "2.1.2"
+    },
+    {
+        "name": "2.1.1",
+        "version": "2.1.1"
+    },
+    {
+        "name": "2.1.0",
+        "version": "2.1.0"
+    },
+    {
+        "name": "2.0.2",
+        "version": "2.0.2"
+    },
+    {
+        "name": "2.0.1",
+        "version": "2.0.1"
+    },
+    {
+        "name": "2.0.0",
+        "version": "2.0.0"
+    },
+    {
+        "name": "2.0.0 (preview)",
+        "version": "2.0.0-preview"
+    },
+    {
+        "name": "1.6.3",
+        "version": "1.6.3"
+    },
+    {
+        "name": "1.6.2",
+        "version": "1.6.2"
+    },
+    {
+        "name": "1.6.1",
+        "version": "1.6.1"
+    },
+    {
+        "name": "1.6.0",
+        "version": "1.6.0"
+    },
+    {
+        "name": "1.5.2",
+        "version": "1.5.2"
+    },
+    {
+        "name": "1.5.1",
+        "version": "1.5.1"
+    },
+    {
+        "name": "1.5.0",
+        "version": "1.5.0"
+    },
+    {
+        "name": "1.4.1",
+        "version": "1.4.1"
+    },
+    {
+        "name": "1.4.0",
+        "version": "1.4.0"
+    },
+    {
+        "name": "1.3.1",
+        "version": "1.3.1"
+    },
+    {
+        "name": "1.3.0",
+        "version": "1.3.0"
+    },
+    {
+        "name": "1.2.2",
+        "version": "1.2.2"
+    },
+    {
+        "name": "1.2.1",
+        "version": "1.2.1"
+    },
+    {
+        "name": "1.2.0",
+        "version": "1.2.0"
+    },
+    {
+        "name": "1.1.1",
+        "version": "1.1.1"
+    },
+    {
+        "name": "1.1.0",
+        "version": "1.1.0"
+    },
+    {
+        "name": "1.0.2",
+        "version": "1.0.2"
+    },
+    {
+        "name": "1.0.1",
+        "version": "1.0.1"
+    },
+    {
+        "name": "1.0.0",
+        "version": "1.0.0"
+    },
+    {
+        "name": "0.9.2",
+        "version": "0.9.2"
+    },
+    {
+        "name": "0.9.1",
+        "version": "0.9.1"
+    },
+    {
+        "name": "0.9.0",
+        "version": "0.9.0"
+    },
+    {
+        "name": "0.8.1",
+        "version": "0.8.1"
+    },
+    {
+        "name": "0.8.0",
+        "version": "0.8.0"
+    },
+    {
+        "name": "0.7.3",
+        "version": "0.7.3"
+    },
+    {
+        "name": "0.7.2",
+        "version": "0.7.2"
+    },
+    {
+        "name": "0.7.0",
+        "version": "0.7.0"
+    }
+]
```

Comment:
Why are there no links for versions below 0.7.0? I found that there are no corresponding pyspark documents for versions below 0.7.0 in the historical directory.
<img width="1168" alt="image" src="https://github.com/apache/spark/assets/15246973/7b599f9c-43cf-4c4d-befc-1dadd40ea04c">
https://spark.apache.org/docs

Reply:
I think that is because we don't have PySpark before 0.7 :)

https://spark.apache.org/releases/spark-release-0-7-0.html

![image](https://github.com/apache/spark/assets/7322292/0d2cf57c-8d16-4d52-b61a-589dfee933f5)